app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,41 @@
+class CategoriesController < ApplicationController
+  include CategoriesHelper
```

Comment:
our helper modules are intended for our views. is there another place we could put the logic that you put here?